### PR TITLE
Add twig lint check and move bash script from tests-legacy to tests

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Add twig lint check and move bash script from tests-legacy to tests
| Type?         | new feature
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | 
| How to test?  | Travis should be green

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/17094)

### CODE REVIEW

**Review by PierreRambaud — DISMISSED** [review]

File: tests/check_file_syntax.sh [hunk 1]
```diff
@@ -10,10 +10,14 @@ yaml_app=$?
 php bin/console lint:yaml themes
 yaml_themes=$?
 
+# Twig
```
Comment (Progi1984) [inline tests/check_file_syntax.sh]: Not the good place (three lines under)

Comment (matks) [reply tests/check_file_syntax.sh]: Nice catch 😄

Comment (matks) [reply tests/check_file_syntax.sh]: Fixed 👍

**Review by PierreRambaud — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Comment by matks** [comment]
Thank you for review @Progi1984 and @PierreRambaud

**Pull request merged**